signalfx-org-metrics/README.md.jinja

```diff
@@ -48,6 +48,13 @@ The `sf.org.numDetectorsAborted` metric monitors the number of detectors that we
 
 For information about events, see [View Additional Data With Events](https://docs.signalfx.com/en/latest/detect-alert/events-intro.html).
 
+### Compare gross and num metric values
+
+Some metrics report a `gross` value and a `num` value. Compare the `gross` and `num` values of a metric to learn about how the system limits or filters data for whatever the metric represents.
+
+A `gross` metric reports the total number of MTS the system receives before any throttling or filtering kicks in. 
```

Comment:
these two lines are incorrectly using MTS. datapoints would be a better noun for those calls but these also exist for spans and rum Spans.